sfge/src/main/java/com/salesforce/graph/symbols/PathScopeVisitor.java

```diff
@@ -1600,7 +1592,8 @@ public boolean visit(VariableDeclarationVertex vertex) {
         final String key = vertex.getName();
         if (apexValueStack.peek().containsKey(key)) {
             // The variable was defined multiple times
-            throw new UnexpectedException(vertex);
+            throw new UserActionException(UserFacingMessages.VARIABLE_DECLARED_MULTIPLE_TIMES,
+                vertex.getFileName(), vertex.getDefiningType(), vertex.getBeginLine());
```

Comment:
Throwing an informative exception that user can act on.

Reply:
I think we handle that scenario. It shouldn't conflict since the symbols are defined in different scopes.